cfme/tests/candu/test_azone_graph.py

```diff
@@ -39,7 +39,9 @@ def azone(appliance, provider):
 @pytest.mark.parametrize("interval", INTERVAL)
 @pytest.mark.parametrize("graph_type", GRAPHS)
 @pytest.mark.meta(
-    blockers=[BZ(1671580, unblock=lambda provider: not provider.one_of(AzureProvider))]
```

Comment:
Instead of removing this BZ, can you move it to an automates meta marker please?

Reply:
@mshriver , Done